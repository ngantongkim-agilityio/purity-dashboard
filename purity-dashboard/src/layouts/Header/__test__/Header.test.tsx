import { render } from '@testing-library/react';

// Components
import { Header } from '..';

describe('Header component', () => {
  it('Should render Header component with default props', () => {
    const { container } = render(<Header />);

    expect(container).toMatchSnapshot();
  });
});
