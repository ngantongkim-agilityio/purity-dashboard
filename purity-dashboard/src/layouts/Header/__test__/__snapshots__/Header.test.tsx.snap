// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Should render Header component with default props 1`] = `
<div>
  <section
    class="flex h-full flex-col px-3 py-4 md:px-2"
  />
</div>
`;
